autobahn/websocket/protocol.py

```diff
@@ -3394,7 +3394,7 @@ def processProxyConnect(self):
             # HTTP version
             #
             http_version = sl[0].strip()
-            if http_version != "HTTP/1.1":
+            if http_version not in ("HTTP/1.1", "HTTP/1.0"):
                 return self.failProxyConnect("Unsupported HTTP version ('%s')" % http_version)
```

Comment:
Once the CONNECT exchange concludes, the protocol is effectively restarted with the destination server, so this doesn't affect the HTTP version of the onward connection.